packages/features/test/unit/index.test.ts

```diff
@@ -39,8 +39,22 @@ test('getCollection() should return a collection', async function() {
 });
 
 test('getFeatures() should fetch features with parameters', async function() {
+  const searchParams = new URLSearchParams();
+
+  // order is important and must match processing in service.getFeatures().
```

Comment:
Just a comment. We can leave this as-is, but in the long term, a test should not hold an insight into the internal logic of the function to test. Or in another word, query parameter order doesn't matter.

Reply:
I agree. I stumbled up on this, because I added the new query parameters to the existing URL and than the test crashed. I think the best aproach would be to use `fetch-mock`'s `query` option.